web/app/mu-plugins/disallow-indexing.php

```diff
@@ -3,12 +3,22 @@
 Plugin Name:  Disallow Indexing
 Plugin URI:   https://roots.io/bedrock/
 Description:  Disallow indexing of your site on non-production environments.
-Version:      1.0.0
+Version:      2.0.0
 Author:       Roots
 Author URI:   https://roots.io/
+Text Domain:  roots
 License:      MIT License
 */
 
-if (defined('WP_ENV') && WP_ENV !== 'production' && !is_admin()) {
+if (!defined('WP_ENV') || WP_ENV === 'production') {
+    return;
+}
+
+if (!is_admin()) {
     add_action('pre_option_blog_public', '__return_zero');
+} else {
+    add_action('admin_notices', function () {
+        $message = __('Search engine indexing is currently discouraged.', 'roots');
+        echo sprintf('<div class="notice notice-warning"><p>%s</p></div>', $message);
```

Comment:
Use `printf()` instead of `echo sprintf()`

Reply:
why use either? why not just inline the var?